pytest_robotframework/__init__.py

```diff
@@ -166,8 +166,8 @@ def inner(*args: P.args, **kwargs: P.kwargs) -> T:
                 StatusReporter(
                     running.Keyword(name=keyword_name, args=log_args),
                     result.Keyword(
-                        kwname=keyword_name,
-                        libname=self.module,
+                        name=keyword_name,
+                        owner=self.module,
```

Comment:
this seems to be the only breaking change that effects us. do we want to continue supporting robot 6? if so we need a version check here and ideally a way to test against both versions in the ci

Reply:
not worth the hassle